src/main/jhipster/src/main/webapp/app/germplasm-manager/inventory/lot-creation-dialog.component.ts

```diff
@@ -52,6 +53,13 @@ export class LotCreationDialogComponent implements OnInit {
         const queryParams = this.activatedRoute.snapshot.queryParams;
         this.searchRequestId = queryParams.searchRequestId;
 
+        if (queryParams.studyId) {
+            // studyId has value if this Lot Creation page is called from Study Manager.
+            // In this case, deposit is required.
+            this.studyId = queryParams.studyId;
+            this.initialDeposit = true;
```

Comment:
I find `initialDeposit` variable name ambiguous. Should it be `initialDepositSpecified`, `initialDepositRequired`?

Reply:
I think initialDepositRequired is more appropriate. Done.